src/browser/actions/makeTestInvokeActions.js

```diff
@@ -30,7 +30,7 @@ const testInvoke = async ({ net, scriptHash, operation, args }) => {
 export default function makeTestInvokeActions(sessionId, requestId) {
   const id = generateDAppActionId(sessionId, `${ID}-${requestId}`);
 
-  return createActions(id, ({ net, scriptHash, operation, args }) => () => {
-    return testInvoke({ net, scriptHash, operation, args });
+  return createActions(id, ({ net, scriptHash, operation, args, encodeArgs }) => () => {
```

Comment:
It looks like `encodeArgs` is being treated as falsy, but is really just `undefined`.  It might be more clearer to specify `encodeArgs = false` here, assuming that's the correct default.

Reply:
We're explicitly passing an argument though, which means the default inside of createScript won't ever be used. That's why I suggest moving/adding it here. :smile:

Reply:
Alright, thanks for explaining! 😄